sdk/src/beta9/abstractions/base/runner.py

```diff
@@ -296,6 +306,7 @@ def prepare_runtime(
 
         if not self.files_synced:
             sync_result = self.syncer.sync()
+            self.remove_tmp_files()
```

Comment:
Shouldn't this be called if it syncs _or_ fails to sync, regardless?

Reply:
I'm calling it before checking whether the sync succeeded or failed. That way in any case it will be removed.